spot-client/src/common/app-state/remote-control-service/actions.js

```diff
@@ -79,3 +104,61 @@ export function setVideoMute(mute) {
                 requestTypes.VIDEO_MUTE, requestStates.ERROR)));
     };
 }
+
+/**
+ * Start wireless screensharing.
+ *
+ * @returns {Function}
+ */
+export function setWirelessScreensharing() {
+    return dispatch => {
+        dispatch(setRequestState(
+            requestTypes.SCREENSHARE,
+            requestStates.PENDING,
+            'proxy'
+        ));
+
+        return remoteControlService.setWirelessScreensharing(true, {
+            onClose: () => dispatch(setLocalWirelessScreensharing(false))
+        })
```

Comment:
is the indentation of `})` correct here ?

Reply:
I like it here and the linter didn't complain. Should it be on line 122?

Reply:
It feels detached, but I had troubles myself what to do in similar situation. You can leave it there if the linter doesn't complain.

Reply:
I tried moving it. So now it's:
```
        return remoteControlService.setWirelessScreensharing(
            true,
            { onClose: () => dispatch(setLocalWirelessScreensharing(false)) }
        )
            .then(() => {
```
Maybe the .then shouldn't be indented? Or maybe it should be placed on the same line as the closing parenthesis.

Reply:
Let me know if it needs more tweaking.